technical-reports/color/token-naming.md

```diff
@@ -0,0 +1,389 @@
+# Token naming
```

Comment:
- [ ] Remove this document? Since we’re updating the format itself, and not providing recommendations (this starts to conflict with other upcoming proposals like modes), we could always split out ideas here into separate proposals or separate conversations to be discussed apart from the color changes.